packages/daemon/src/runner/helper.ts

```diff
@@ -13,8 +14,13 @@ import {
 
 const { PLUGINS, PIPCOOK_LOGS } = constants;
 
-export async function parseConfig(configPath: string, generateId = true): Promise<PipelineDB> {
-  const configJson: RunConfigI = await fs.readJson(configPath);
+export async function parseConfig(config: string, generateId = true): Promise<PipelineDB> {
```

Comment:
The function signature should not be changed :)

Reply:
But the meaning of `configPath` has actually changed.

Reply:
It acts as the pathname for pipeline config.

Reply:
OK.